typescript/quantize/lab_point_provider.ts

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-import './point_provider';
+import { PointProvider } from './point_provider.js';
```

Comment:
Now that all the local imports/exports use `.js` extensions, experimental specifiers are no longer needed/will no longer do anything when running this library.
Would it make sense to remove the [troubleshooting section in the readme](https://github.com/material-foundation/material-color-utilities/blob/main/typescript/README.md#troubleshooting)?

Reply:
Yeah good call out!